UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -0,0 +1,98 @@
+﻿using HealthV2;
+using Mirror;
+using Player.Movement;
+using UnityEngine;
+
+namespace Player
+{
+	public class LayDown : NetworkBehaviour
+	{
+		[SerializeField] private Transform sprites;
+		[SerializeField] private LivingHealthMasterBase health;
+		[SerializeField] private Rotatable playerDirectional;
+		[SerializeField] private PlayerScript playerScript;
+		[SerializeField] private Util.NetworkedLeanTween networkedLean;
+		private readonly Quaternion layingDownRotation = Quaternion.Euler(0, 0, -90);
+		private readonly Quaternion standingUp = Quaternion.Euler(0, 0, 0);
+
+		[SyncVar(hook = nameof(OnLayDown))] public bool IsLayingDown = false;
+
+
+		private void Awake()
+		{
+			playerScript ??= GetComponent<PlayerScript>();
+			playerDirectional ??= GetComponent<Rotatable>();
+			health ??= GetComponent<LivingHealthMasterBase>();
+		}
+
+		private void OnEnable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Add(EnsureCorrectState, 5f);
+			}
+		}
+
+		private void OnDisable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, EnsureCorrectState);
+			}
+		}
+
+		public void EnsureCorrectState()
+		{
+			if (IsLayingDown || health.IsDead) { LayingDownLogic(true); } else { UpLogic(true); }
```

Comment:
please do a single line per block. The body of each if and else should have their own line to live on.

Reply:
> > We're moving away from whole gameObject rotations and are only rotating the sprites from now on.
> 
> This should have been the case from the very beginning and it was like that before, dunno when we changed to move the game object again. Anyways, thanks for doing this.

It was changed when Bod was working on the new movement to support players having these funny rotation moments when getting thrown around

Reply:
done